fontgenerator.js

```diff
@@ -433,8 +431,9 @@ TinyFontGenerator = {
     var data = ""; 
     data += " 00 01 00 00";  // version 1
     data += " 00 00 00 00";  // italicAngle
-    data += " 00 00 00 00";  // underlinePosition
-    data += " 00 00 00 00";  // underlineThickness
+    data += " 00 00";        // underlinePosition
```

Comment:
btw, I didn't change anything in the post table, I simply reformatted the string so that it shows that `isFixedPitch` field is actually in there, whereas previously it appeared as if it wasn't.